fbmessenger/__init__.py

```diff
@@ -88,6 +88,19 @@ def delete_get_started(self):  # pragma: no cover
         )
         return r.json()
 
+    def delete_persistent_menu(self):  # pragma: no cover
```

Comment:
Can you add a test for this, you can use `test_delete_get_started` from `test_client.py` as a reference